src/main/resources/db/migration/V1.0__create_items_table.sql

```diff
@@ -0,0 +1,22 @@
+CREATE TABLE resources(
+    item_id         BIGINT            NOT NULL,
```

Comment:
nitpick:
```suggestion
    resource_id         BIGINT            NOT NULL,
```

or just `id`